sdcm/remote/base.py

```diff
@@ -233,19 +233,20 @@ def __init__(self, log_file: str):
         super().__init__()
         self.len = 0
         self.log_file = log_file
+        # open fail with line buffering, so prom stats would be as accuracte as possible
+        # pylint: disable=consider-using-with
+        self.file_object = open(self.log_file, "a+", encoding="utf-8", buffering=1)
```

Comment:
If we are leaking this object, then we might leak a file descriptor, until the end of the run.
but we use this mostly for stress commands, if they end, this would be cleared.